i18n/es/translation.json

```diff
@@ -479,5 +479,30 @@
   "There is no voting power in the connected wallet, and no votes have been delegated to you.": "No hay poder de voto en la billetera conectada, y no se te han delegado votos.",
   "Delegated to you": "Delegado a ti",
   "On connected wallet": "En billetera conectada",
-  "Delegated to": "Delegado a"
+  "Delegated to": "Delegado a",
+  "Transaction Summary": "Resumen de la Transacción",
+  "Estimated Output": "Salida Estimada",
```

Comment:
what do you think of "Retiro estimado"?

Reply:
I'll use 'Monto' because we're talking about how much EXA you're going to get